### DISCUSSION TITLE
UnicodeDecoeError in example markdown.py

### DISCUSSION THREAD

**Discussion by Jimmy-KL** [Kitware/trame#162] [Q&A] [answered]
I tried to run example [01_markdown.py](examples/03_markdown/01_markdown.py) , but I got the following error. I don't know whether it's a bug. Anyway, please take a look.
![image](https://user-images.githubusercontent.com/114913030/205491704-dd4ce0da-f624-4b97-98a6-01e883bc7baa.png)

**Comment by jourdain** [comment, marked as answer]
[This commit](https://github.com/Kitware/trame/commit/2760bc28917fa31cc7f97ea8dd9a361cfe9f541c) should solve your issue by providing encoding at read time.
